store/tikv/scan.go

```diff
@@ -230,7 +263,11 @@ func (s *Scanner) getData(bo *Backoffer) error {
 		// may get an empty response if the Region in fact does not have
 		// more data.
 		lastKey := kvPairs[len(kvPairs)-1].GetKey()
-		s.nextStartKey = kv.Key(lastKey).Next()
+		if !s.desc {
+			s.nextStartKey = kv.Key(lastKey).Next()
+		} else {
+			s.nextEndKey = kv.Key(lastKey)
```

Comment:
You need to use lastKey.Prev() to locate the next region. But you will never know what is the prev key of the certain key. So I think you should use GetPrevRegion to find out what region is the next one. Ref https://github.com/pingcap/pd/issues/1055